site/ja/tutorials/keras/save_and_restore_models.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<blockquote>重みの値、モデルの設定、（構成によっては）オプティマイザの設定までも含んだモデル全体をファイルに保存することができます。これにより、モデルのチェックポイントを保存し、オリジナルのコードにアクセスする必要なく、中断したところから訓練を再開することができます。</blockquote><p><br></p><p>"チェックポイント" can be misunderstood as  .ckpt file. How about:</p><p><br></p><p>モデルとオプティマイザは、それらの状態（重みと変数）とモデル構成の両方を含む1つのファイルに保存することができます。これにより、モデルをエクスポートして、オリジナルのPythonコードにアクセスしなくても使用できるようになります。オプティマイザの状態についても復元されるので、中断したところから訓練を再開することもできます。</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/390/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
I'd like to change this paragraph as

重みの値、モデルの設定、そして（構成によりますが）オプティマイザの設定までも含んだモデル全体をファイルに保存することができます。これにより、モデルのある時点の状態を保存し、オリジナルのPythonコードにアクセスしなくとも、中断したところから訓練を再開することができます。

As the optimizer's configuration can not to be saved in a hdf5 file, if tf.train optimizer used.

Reply:
OK. I understand. Looks good to me. But..
"構成によりますが" means "if tf.keras.optimizers optimizers are used", right?
If so, it may be better to write explicitly  or add "訳注 (translator's note)".

Reply:
Thanks for your suggestion.
I'd like to add a translator's note.